script/Ciel/Day 13/13_01_02_CIEL13_7.txt

```diff
@@ -316,7 +362,9 @@ But that's okay. No matter what she throws my way, I'll definitely overcome it.
 [sha:02414c6a9c056d23dcf99f7fe59ce5acdee6e785]{
 -- Page 223, Offset 35901.
 -- そんなコトより、ここでこいつを口にしないと、俺の<原|こ><動|こ><力|ろ>が消えそうだ。
-More importantly, if I don't voice my thoughts now, I feel like my <drive|m i n d > is going to fade away.
+More importantly,
+ If I don't say what's on my mind now,
+ I feel like my <motivation|e m o t i o n s   > will dry up.
```

Comment:
こころ here is probably because like, "otherwise Roa will take over" kinda deal. Mind is more appropriate, but it doesn't work with the verbal structure.

>I feel like I'll lose my <motivation|mind>.